content/docs/09-registries-and-packs/4.5-create-pack.md

```diff
@@ -0,0 +1,718 @@
+Use the following values in the first section, **Basic Information**. 
+
+|**Field**|**Value**|
+|---|---|
+|Cluster name| *pack-tutorial-cluster* |
+|Description| *Cluster as part of the pack tutorial.*|
+|Tags|`spectro-cloud-education, app:hello-universe, terraform_managed:true`|
+|Cloud Account|**spectro-cloud**|
+
+Note that the value **spectro-cloud** is the AWS cloud account name already configured in the Palette project settings. 
+
+<br /> <br />
+
+### Section: Parameters
+Next, the **Parameters** section allows you to change the profile configurations. For example, clicking on the **Hello Universe 1.0.x** layer will allow you to configure the `registry`, `repository`, and `tag` parameters defined in the **values.yaml** file. Let's keep those parameter values as default and move to the next section. 
+
+<br /> <br />
+
+### Section: Cluster config
+In the **Cluster config** section, uncheck the **Static Placement** field, select the **us-east-2** region, and select the SSH key pair name you have in your AWS console. If checked, the **Static Placement** will deploy the cluster in an existing VPC, subnets, roles, and other resources. But, for this tutorial, you will choose the dynamic placement, meaning the Palette dev engine will create a new VPC and all other resources needed for the cluster. 
+
+<br /> <br />
+
+### Section: Nodes config
+In the **Nodes config** section, provide the details for the master and the worker pool. For learning purposes, you can use the following configuration:
+
+|**Field** | **Value for the master-pool**| **Value for the worker-pool**|
+|---| --- | ---|
+|Node pool name| *master-pool* | *worker-pool* |
+|Number of nodes in the pool| `1` | `1` |
+|Allow worker capability| *Checked* | Not applicable |
+|Enable Autoscaler | Not applicable | No |
+|Rolling update |  Not applicable | *Expand* <br /> Launch a new node first, then shut down the old one. |
+|Instance Type | General purpose `m4.large` | Copy from Master pool | 
+|Availability zones | **us-east-2a** <br />Choose only one availability zone. | Copy from Master pool | 
+|Disk size | 60 GiB | Copy from Master pool | 
+
+<br /> <br />
+
+### Section: Settings and Review
+Use the default values in the **Settings** section and review all configurations in the **Review** section.  If everything looks good, finish creating the cluster configuration. It will start deploying the cluster and may take up to 15-20 minutes to finish.
+
+<br />
+
+![Visual of the cluster health.](/tutorials/create-pack/registries-and-packs_create-pack_cluster-health.png)
+
+<br /> <br />
+
+You can also look at real-time metrics on the **Overview** tab, such as CPU and memory consumption. 
+
+<br /> <br />
+
+## Validate
+
+After the Palette finishes provisioning the cluster, the status will turn to **Running** and **Healthy**. You can access the application on the exposed port 8080 for the **Hello Universe** service. The application will look similar to the visual below. 
```

Comment:
```suggestion
After Palette finishes provisioning the cluster, the status will turn to **Running** and **Healthy**. You can access the application from the exposed service URL with the port number displayed. For the Hello Universe application, port 8080 is exposed. Click on the URL to access the application.
```

We don't refer to Palette as "The Palette", just Palette. Like Facebook, there is no "The Facebook". It's just "Facebook". 

Also, due to DNS behavior, it might be wise to add a warning to the user to wait a min or two.


![CleanShot 2023-04-21 at 17 14 54](https://user-images.githubusercontent.com/29551334/233751362-4fbc43dd-d10b-456b-b903-0c8b1c740e2a.png)